app/models/concerns/item_categorization.rb

```diff
@@ -1,25 +1,13 @@
 require "active_support/concern"
 
-module ItemAttributes
+module ItemCategorization
```

Comment:
The story here is that the category code is reusable in both `Item` and `ItemPool`, but `power_source` isn't now, so I made this file only include the category parts.